### ISSUE TITLE
Investigate spaces around operators

### ISSUE THREAD

**Issue by krzysztofrewak** [blumilksoftware/codestyle#26]
Here https://github.com/blumilksoftware/eupms/pull/202/commits/69cb96f7bd21da5d327f49a755a8c1969f20bbf4 I noticed that ECS didn't throw any error on this:

```php
$userId=$auth->user()->id;
```

It definitely should has spaces before and after the operator.
